problemsets-mat223/definitions.tex

```diff
@@ -158,25 +159,27 @@
 	title={Linearly Dependent \& Independent (Geometric)}]
 
 	We say the vectors $\vec v_{1},\vec v_{2},\ldots,\vec v_{n}$ are
-	\emph{linearly dependent} if for at least one $i$,
+	\emph{linearly dependent}\index{Linearly dependent}\index[definitions]{Linearly dependent}\index[definitions]{Linearly dependent!(geometric)} if for at least one $i$,
```

Comment:
- > `Linear independence/dependence!geometric definition`. Put `Linear independence/dependence` rather than one or the other.